operations/messaging-and-math/content-marketing/video-content.md

```diff
@@ -0,0 +1,76 @@
+* **Don’t be afraid of do-overs:** if you’re recording something specifically for video distribution (not live) and something happens -- a police car goes by outside, cat knocks over your mic, you say the wrong word -- feel free to stop, take a breath, and redo that section. 
+* **Take notes:** If your co-speaker mentions a customer that we shouldn’t discuss publicly, or you needed to do-over on something, time-stamped notes are helpful for making the editing process faster and easier. 
+
+*An additional note for recording webinars, meetings and other large-group calls: if there are more people than the speakers on the call being recorded, have them mute their mics and turn off their webcams before the recorded section begins. You can opt to hide participants who are not visible in your Zoom settings. This helps keep the recording free of visual and audio clutter.* 
+
+## Post-Production/Video Processing Workflow
+### What to prepare: 
+* A description of the video
+* Raw video(s) for editing
+* Transcript (if available)
+* Title and end cards (as needed)
+* Notes
+    * Where does the video start? Where does it end? 
+    * Do multiple videos need to be edited together?
+    * Are there any sections that need to be removed? 
+    * Are there any sound or video quality issues that should be checked?
+
+### Submitting to Fiverr
+For video editing we use Fiverr vendors to prepare polished videos. The following are our preferred vendors: 
+* Watchclosely
+* Graphifyme
+* matteo505
+
+### Reviews & Edits
+Minimize the number of edits and stakeholders for any given video, and be prompt with giving feedback to Fiverr editors. Fiverr tickets close automatically after 3 days, so be mindful of getting your feedback in quickly. 
+
+Editing should include:
+* Editorial owner
+* (Optional) Internal stakeholders, e.g. a developer who gave a presentation on an upcoming feature 
+* (Optional) External stakeholders, e.g. a customer who participates in a video case study
+
+## Preparing Finished Videos for Publication
+
+Mattermost videos are hosted on Youtube, which has a very simple upload workflow. Upload the video via Youtube Studio and include the following metadata:
+* Thumbnail image
+* Video title
+* Video description
+* Tags
+
+If video is going to be available on our Youtube channel, assign it to appropriate playlists. If video will not be available on Youtube (e.g. a customer testimonial that will only be embedded on the Customers page), do not assign it to playlists and set state to “Unlisted” (not “Private”).
+
+For access to our Youtube channel, ping @hanna.park.  
+
+## Video Content Distribution  
+
+Video content may be promoted as part of a blog post, guide or other larger content campaign, or it may require its own dedicated campaign. Follow best practices for our [Content Distribution process](https://handbook.mattermost.com/operations/messaging-and-math/content-marketing/content-development-distribution-processes). 
```

Comment:
```suggestion
Video content may be promoted as part of a blog post, guide or other larger content campaign, or it may require its own dedicated campaign. Follow best practices for our [Content Distribution process](https://handbook.mattermost.com/operations/messaging-and-math/content-marketing/content-development-distribution-processes).
```